changelog.d/issue-8785

```diff
@@ -0,0 +1,29 @@
+synopsis: Also render short options with arguments
+packages: cabal-install
+prs: #9043
+issues: #8785
+
+description: {
+
+Show how arguments are used with both short and long forms of options:
+
+```diff
```

Comment:
I think this whole diff in a changelog file is a little excessive. One example should suffice. The place I'd like to have all these examples are commit message.

Reply:
I shortened the diff to two representative types.